docs/003-packaging.md

```diff
@@ -150,43 +149,41 @@ As you might have noticed, in the `package-and-publish` job in the workflow file
 
 ### Limits of the GITHUB_TOKEN
 
-Note that the permissions that can be granted to the `GITHUB_TOKEN` are limited to the scope of the repository of the Actions Workflow itself. That suffices for a lot of use-cases - but sometimes, you will want to access and / or change something in another repository or even on the organization level.
-
-This scenario is out of scope for this workshop, but if you are interested in how to solve this, you have two options:
-
-1. Create a [Personal Access Token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with elevated permissions that you then provide to the GitHub Workflow by [storing it in a Repository Secret](https://docs.github.com/en/enterprise-cloud@latest/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository)
-2. Create and install a [GitHub Application](https://docs.github.com/en/enterprise-cloud@latest/apps/maintaining-github-apps/installing-github-apps) in your organization and then use the [Workflow Application Token Action](https://github.com/peter-murray/workflow-application-token-action) to generate a short-lived token during the workflow run
+Note that the permissions that can be granted to the `GITHUB_TOKEN` are limited to the scope of the repository where the Actions workflow is running. While this is sufficient for many use cases, there are times when you might want to access or modify something in another repository or even at the organization level.
 
-## 5 - Find your image in GitHub Container Registry
+This scenario is beyond the scope of this workshop, but if you're interested in addressing this, you have two options:
 
-1. Go to your project main page
+1. Create a [personal access token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with the necessary permissions, and then provide it to the GitHub Actions workflow by [storing it as a repository secret](https://docs.github.com/en/enterprise-cloud@latest/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+2. Create and install a [GitHub App](https://docs.github.com/en/enterprise-cloud@latest/apps/maintaining-github-apps/installing-github-apps) in your organization, and then use the [workflow application token action](https://github.com/peter-murray/workflow-application-token-action) to generate a short-lived token during the workflow run.
 
-2. Click on the **Packages** link in the right menu
+## 5 - Locate your image in the GitHub Container Registry
 
-3. Click on your container
+1. Navigate to your project's main page.
+2. Click on the **Packages** link on the right menu.
+3. Select your container.
 
 ![](../images/img-037.png)
 
 ## Conclusion
 
-In this lab you have learned how to :
+In this lab, you've learned how to:
```

Comment:
Do we really want to change to "you've"?

Reply:
I'd leave it in the long form.

```suggestion
In this lab, you have learned how to:
```